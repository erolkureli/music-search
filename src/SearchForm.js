import React from 'react';
import { SearchFormRow, Label } from './StoreSearch';

function SearchForm(props) {

    return (
        <>
            <SearchFormRow>
                <Label>
                    Search Type : 
                </Label>
                <select value={props.searchType} onChange={props.onSearchTypeChange}>
                    <option value="musicArtist">Music Artist</option>
                    <option value="album">Album</option>
                    <option value="song">Song</option>
                </select>
            </SearchFormRow>
            <SearchFormRow>
                <Label>
                    Search Text :
                </Label>
                <input type="text" value={props.searchText} onChange={props.onSearchTextChange} />
            </SearchFormRow>
        </>
    );
}

export default SearchForm;