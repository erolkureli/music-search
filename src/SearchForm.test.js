import React from 'react';
import { configure , shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });



import SearchForm from './SearchForm';

describe('SearchForm', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<SearchForm debug />);
  
    expect(component).toMatchSnapshot();
  });
});
