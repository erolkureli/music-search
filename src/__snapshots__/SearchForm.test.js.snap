// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
